Makefile

```diff
@@ -13,6 +15,9 @@ lint:
 test_rules:
 	./automation/promtool_tests.sh test_rules
 
-.PHONY: pint
 pint:
-	./automation/pint_lint.sh
```

Comment:
Are we still running the tests?

Reply:
@yftacherzog There's still an issue here I'm trying to resolve..

Reply:
Yes. I think we need them separated into 2 targets. One will be called `pint` and will install it and another will be phony and will have a different name (I'm sure that there are ways to tell make to consider binaries with different names than the target name, but I'm not familiar enough with that).

I'd say the installation needs to be a target which will be a dependency for the `prepare` target (later on we can have the other parts of `prepare` as targets which will be dependencies for `prepare` as well), and then running the tests can happen as another target (with a name which is not `pint`).

Reply:
Something like this:
```
all: prepare lint test_rules run_pint

prepare: pint
     ./automation/promtool_tests.sh prepare (we'll change this on a later PR as well and turn it into another dependency)

pint:
	echo 'Installing pint...'
	curl -OJL 'https://github.com/cloudflare/pint/releases/download/v${PINT_VERSION}/pint-${PINT_VERSION}-linux-amd64.tar.gz'
	tar xvzf 'pint-${PINT_VERSION}-linux-amd64.tar.gz' pint-linux-amd64
	mv pint-linux-amd64 pint
	rm 'pint-${PINT_VERSION}-linux-amd64.tar.gz'

.PHONY:  run_pint
run_pint:
        ./automation/pint_lint.sh
```

Reply:
Temporary in what sense? You mean until we make it all part of an image?

Reply:
@yftacherzog Yup

Reply:
:crossed_fingers: